packages/core/docs/integrations/available-integrations.md

```diff
@@ -0,0 +1,17 @@
+# Available integrations
```

Comment:
what this document is shoudl describe and what si the value of publishing a document with only todos?